@AH-64D Apache Longbow/Addons/fza_ah64_wca/pushbuttons/emerHyd.hpp

```diff
@@ -23,18 +23,22 @@ class Draw {
     #define CHAR_WIDTH_VEC 0.32
     #define CHAR_HEIGHT 0.5
 
-    class pushButton {
-        class emer_hyd {
-            type = "text";
-            source = "static";
-            text = "ON";
-            sourceIndex = 0;
-            scale=1.0;
-            align="center";
-            sourceScale = 1;
-            pos[]   = {{START_X,                  START_Y - 0.5*CHAR_HEIGHT}, 1};
-            right[] = {{START_X + CHAR_WIDTH_VEC, START_Y - 0.5*CHAR_HEIGHT},1};
-            down[]  = {{START_X,                  START_Y - 0.5*CHAR_HEIGHT + CHAR_HEIGHT},1};
+    class battOff {
+        condition = C_COND(C_USER(MFD_IND_BATT));
```

Comment:
You can merge these two into a single condition

```cpp
condition = C_COND(C_AND(C_USER(MFD_IND_BATT), C_USER(MFD_IND_EMER_HYD)));
```

Reply:
Fixed